app/packages/state/src/hooks/useClearSessionColorScheme.ts

```diff
@@ -19,17 +20,31 @@ const useClearSessionColorScheme = () => {
   const setSessionColorSchemeState = useSetRecoilState(sessionColorScheme);
   const dataset = useRecoilValue(datasetName);
   const stages = useRecoilValue(view);
+  const defaultSetting = useRecoilValue(datasetAppConfig).colorScheme;
 
   function onClear(saveToApp: boolean) {
```

Comment:
`useMemo` could be good to use here

Reply:
"useRecoilValue" cannot be called inside the useMemo callback.